application-analytics-default/pom.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.xwiki.analytics</groupId>
+    <artifactId>application-analytics</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>application-analytics-default</artifactId>
+  <packaging>jar</packaging>
+  <name>Analytics Application (Pro) - DEFAULT</name>
+  <description>Discover unique insights to help you better understand the usage of your instance. Streamline and improve
+    internal processes by having full visibility on the most viewed pages. Understand how you can help your teams boost
+    productivity by using real-time statistics.
+  </description>
+  <properties>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>com.xwiki.analytics</groupId>
+      <artifactId>application-analytics-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>javax.ws.rs</groupId>
+      <artifactId>jsr311-api</artifactId>
+    </dependency>
+    <!-- Testing dependencies -->
+    <!-- UriBuilder in JAX-RS is abstract implementation and requiring an implementation like Jersey for unit testing.
```

Comment:
I'd put this before the 2 jersey dependencies, instead of the `<!-- Jersey core libraries -->`